static-assets/themes/cstudioTheme/css/forms-default.css

```diff
@@ -1532,12 +1563,17 @@ body.yui-skin-cstudioTheme.masked {
   margin-left: 5px;
 }
 
-.cstudio-form-control-node-selector-controlbox {
+.cstudio-form-control-node-selector-controlbox,
+.cstudio-form-control-transcoded-video-controlbox {
   width: 100%;
   display: flex;
   flex-wrap: wrap;
 }
 
+.cstudio-form-control-transcoded-video-controlbox {
+  /*.cstudio-*/
+}
+
```

Comment:
?

Reply:
The selector was going to be used for a specific style but it wasn't at the end. Removing